tests/gold/cmd-fleet-recovery-test/050-export.txt

```diff
@@ -1,6 +1,6 @@
 # Export the recovery information
-# [fleet, recovery, export, --directory, TMP_DIR]
-Exported recovery information to 'TMP_DIR/recover--={|~~~~~~~~~~FLEET_ID~~~~~~~~~~|}=-.json'.
+# [fleet, recovery, export, -o, TMP_DIR/recovery.json]
+Exported recovery information to 'TMP_DIR/recovery.json'.
```

Comment:
The discrepancy between the output-file name (`recovery.json`) and the server URL (3 lines down) is OK. We have set the recovery URLs previously to just use `recover.json`.